### PR TITLE
refactor: s/command/msg_type/ in CNetMsgMaker and CSerializedNetMsg

### PR DESCRIPTION
**Pull request by theStack**
Follow-up PR for #18533 -- another small step towards getting rid of the confusing "command" terminology. Also see PR #18610 which tackled the functional tests.

### CODE REVIEW

File: src/net.h [hunk 1]
```diff
@@ -108,7 +108,7 @@ struct CSerializedNetMsg
     CSerializedNetMsg& operator=(const CSerializedNetMsg&) = delete;
 
     std::vector<unsigned char> data;
-    std::string command;
+    std::string msg_type;
```
Comment (MarcoFalke) [inline src/net.h:111]:
```suggestion
    std::string m_type;
```

According to the dev notes, members should be prefixed with `m_`.

Comment (theStack) [reply src/net.h]: Thanks, done.

**Comment by MarcoFalke** [comment]
ACK 51e9393c1f6c9eaac554f821f5327f63bd09c8cf

**Comment by jonatack** [comment]
Concept ACK, `msg_type` is better

**Pull request merged**